python/services/dataservice/dmod/dataservice/service_settings.py

```diff
@@ -0,0 +1,19 @@
+from pydantic import BaseSettings, Field
+from typing import Optional
+from typing_extensions import Annotated, TypeAlias
+
+Port: TypeAlias = Annotated[int, Field(..., ge=0, le=65535)]
+
+
+class ServiceSettings(BaseSettings):
+    s3fs_url_protocol: str = "http"
+    s3fs_url_host: Optional[str]
+    s3fs_url_port: Port = 9000
+    s3fs_vol_image_name: str = "127.0.0.1:5000/s3fs-volume-helper"
+    s3fs_vol_image_tag: str = "latest"
+    s3fs_plugin_alias: str = "s3fs"
+    s3fs_helper_network: str = "host"
+
+
+    class Config(BaseSettings.Config):
+        frozen = True
```

Comment:
I don't know what Pydantic > v1 does by default, but should we explicitly include `case_sensitive = False` here?

Reply:
Yep, great idea! I would rather err on the side of being explicit to make our lives easier whenever we inevitably need to port.

Reply:
@robertbartel, just added the change, rebased, and force pushed.